ui/app/controllers/settings/tokens.js

```diff
@@ -14,14 +14,13 @@ export default class Tokens extends Controller {
   @service token;
   @service store;
   @service router;
-  @service flashMessages;
 
   queryParams = ['code', 'state'];
 
   @reads('token.secret') secret;
 
-  tokenIsValid = false;
-  tokenIsInvalid = false;
+  successfulSignIn = false;
+  unsuccessfulSignIn = false;
```

Comment:
`question`: I'm noticing that these properties are wired together. And complimentary opposites. Could it be possible for this to be combined into a field that is an enum? Potentially:

`signInStatus: 'success' | 'failed' | 'idle'`

Also should this be a tracked property that updates reactively?

Reply:
Yeah, this is a good point. There's not much stopping us from updating this, so I think I will modernize it. Thanks for the suggestion!